src/api/java/com/minecolonies/api/inventory/container/ContainerCrafting.java

```diff
@@ -46,6 +48,11 @@
      */
     private final Slot craftResultSlot;
 
+    /**
+     * The secondary outputs
+     */
+    private final java.util.List<ItemStack> remainingItems;
```

Comment:
java.util.List?

Reply:
Yes, just List resolves to something else here.

Reply:
Ok, I know what happened, and I have a fix.